configs/panoptic_fpn/panoptic_fpn_r50_fpn_1x_coco.py

```diff
@@ -0,0 +1,147 @@
+_base_ = [
+    '../_base_/datasets/coco_panoptic.py',
```

Comment:
Should we inherit from a standard faster-rcnn

Reply:
It is better not to inherit because It is not convenient to read.

And the models will be released in a new PR, so it can be discussed in the next PR.